File: GetResourcesAPI/app/src/main/java/com/example/getresourcesapi/MainActivity.java
package com.example.getresourcesapi;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private EditText etValue;
    private Button btnFetch;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        etValue = findViewById(R.id.etNumber);
        btnFetch = findViewById(R.id.btnFetch);
        btnFetch.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, SecondActivity.class);
                int num = Integer.parseInt(etValue.getText().toString());
                intent.putExtra("number", num);
                startActivity(intent);
            }
        });
    }
}

File: Kotlin Projects/MoneyManager/app/src/main/java/com/example/moneymanager/MainActivity.kt
package com.example.moneymanager

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Handler().postDelayed({
            val splashIntent = Intent(this@MainActivity, MoneyManager::class.java)
            startActivity(splashIntent)
            finish()
        }, 3000)
    }
}